src/main/kotlin/no/nav/familie/dokument/storage/attachment/FlattenPdfService.kt

```diff
@@ -1,22 +1,31 @@
 package no.nav.familie.dokument.storage.attachment
 
 import org.apache.pdfbox.pdmodel.PDDocument
+import org.apache.pdfbox.pdmodel.encryption.InvalidPasswordException
 import org.apache.pdfbox.pdmodel.interactive.form.PDAcroForm
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import java.io.ByteArrayOutputStream
 
 
 @Service
 class FlattenPdfService {
 
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
     fun convert(input: ByteArray): ByteArray {
-        val loadedPdf = PDDocument.load(input)
-        val pdAcroForm: PDAcroForm? = loadedPdf.documentCatalog.acroForm
-        return pdAcroForm?.let {
-            it.flatten()
-            val byteArrayOutputStream = ByteArrayOutputStream()
-            loadedPdf.save(byteArrayOutputStream)
-            byteArrayOutputStream.toByteArray()
-        } ?: input
+        return try {
+            val loadedPdf = PDDocument.load(input)
+            val pdAcroForm: PDAcroForm? = loadedPdf.documentCatalog.acroForm
```

Comment:
Tror man kan sjekke om den er encryptet, men dette virker nog også :) 
```kotlin
loadedPdf.isEncrypted()
```

Reply:
Joda, men den feilet vel på `PDDocument.load(input)` - så da er vel det for sent?